setup.cfg

```diff
@@ -3,3 +3,6 @@ current_version = 0.2.0
 commit = True
 
 [bumpversion:file:lang_pref_middleware/__init__.py]
+
+[tool:pytest]
+python_files = test*.py
```

Comment:
why we need `python_files` directive?

Reply:
Without it, pytest was not collecting tests written in test file (https://github.com/edly-io/django-lang-pref-middleware/blob/master/lang_pref_middleware/tests.py).